case-lib/lib.sh

```diff
@@ -364,3 +364,31 @@ logger_disabled()
         return 1
     fi
 }
+
+enable_devres_debug_log()
+{
+    local devres_log
+    devres_log=$(cat /sys/module/devres/parameters/log)
+    if [ "$devres_log" -eq 1 ]; then
+        dlogi "DEVRES log is already enabled"
+        return 0
+    fi
+
+    #TODO: below command didn't work, wrongly overidden and cause permission issue
+    #sudo bash -c "echo 1 > /sys/module/devres/parameters/log"
+    echo 1 | sudo tee /sys/module/devres/parameters/log > /dev/null
+}
+
+disable_devres_debug_log()
+{
+    local devres_log
+    devres_log=$(cat /sys/module/devres/parameters/log)
+    if [ "$devres_log" -eq 0 ]; then
+        dlogi "DEVRES log is already disabled"
+        return 0
+    fi
+
+    #TODO: below command didn't work, wrongly overidden and cause permission issue
+    #sudo bash -c "echo 0 > /sys/module/devres/parameters/log"
```

Comment:
Strange, I prefer the version with `tee` but I tried the other version and it works for me. What does "wrongly overridden" mean?

I don't find this TODO very useful. It doesn't explain why the first command did not work for you and I don't think there's really anything TODO either. If any I'd rather have a comment that explains why the naive `sudo cmd > /sys` does not work.